src/main/java/seedu/duke/Parser.java

```diff
@@ -86,16 +86,19 @@ public static Command parse(String inputCommand) throws DukeExceptions {
 
         boolean isHomeView = (projectIndex == -1); //In main project list view
 
-        Command command = getCommand(isHomeView, commandType, params, projectIndex);
+        Command command = getCommand(isHomeView, commandType, params, projectIndex, inputWords);
         return command;
     }
 
     public static Command getCommand(boolean isHomeView, String commandType, HashMap<String, String> params,
-                                     int projectIndex) throws DukeExceptions {
+                                     int projectIndex, String[] inputWords) throws DukeExceptions {
```

Comment:
hi sam, what is `inputWords`

Reply:
The bug I am trying to solve over here is that, users can enter "list hello everybody" or "list p//" and it still works.
Since our list command does not require any params to execute, there is a need to check for any words behind "list".
inputWords is used in the function before calling getCommand() which checks if there are more than 1 word input.

Reply:
interesting